src/states/elder/mod.rs

```diff
@@ -2030,6 +2044,12 @@ impl Approved for Elder {
         Ok(())
     }
 
+    fn handle_prune(&mut self) -> Result<(), RoutingError> {
+        let _sec_info = NetworkEvent::SectionInfo(self.chain.prune()?);
```

Comment:
Why are we creating a NetworkEvent we do not use?

Reply:
Yeah, I shall remove it. A left over from the previous approach